homeassistant/components/homeassistant/scene.py

```diff
@@ -139,16 +140,21 @@ def _process_scenes_config(hass, async_add_entities, config):
         return
 
     async_add_entities(
-        HomeAssistantScene(hass, SCENECONFIG(scene[CONF_NAME], scene[CONF_ENTITIES]))
+        HomeAssistantScene(
+            hass,
+            scene.get(CONF_ID),
+            SCENECONFIG(scene[CONF_NAME], scene[CONF_ENTITIES]),
+        )
         for scene in scene_config
     )
 
 
 class HomeAssistantScene(Scene):
     """A scene is a group of entities and the states we want them to be."""
 
-    def __init__(self, hass, scene_config):
+    def __init__(self, hass, scene_id, scene_config):
         """Initialize the scene."""
+        self._id = scene_id
```

Comment:
Can we start using this as unique ID too ?

Reply:
We could, but would make naming in the frontend more complicated as we have to take the entity registry in to account. What would be the benefit of an unique id here?

Reply:
guess just custom entity IDs, but you're right, let's leave it for now.